README.md

```diff
@@ -128,3 +128,15 @@ You can set `electronPackagerConfig` with **any** of the options from
 **NOTE:** The `afterCopy` and `afterExtract` options are mapped to `require`
 calls internally, so provide a path to a file that exports your hooks and they
 will still run.
+
+**NOTE:** You can also set your `forge` config property of your package.json to point to a JS file the exports the config object like so.
```

Comment:
* `s/object like so./object:/`
* I would move this note above the hook note, and then adjust the hook note to mention that hook functions are allowed in the JS file version.